tests/wpt/web-platform-tests/xhr/formdata.htm

```diff
@@ -56,8 +56,8 @@
       formDataInEvent = e.formData;
     });
     let formData = new FormData(form);
-    assert_equals(formData.get('h1'), 'vh1');
-    assert_equals(formData.get('n1'), 'v1');
+    assert_equals(formDataInEvent.get('h1'), 'vh1');
+    assert_equals(formDataInEvent.get('n1'), 'v1');
```

Comment:
I cannot pass the original test case locally and then I found it might be more reasonable to test with `formDataInEvent` here? because it's assigned with the appended `e.formData` which should have `h1`.

Reply:
This change should not be necessary if my earlier comment is addressed.